pandas_datareader/_utils.py

```diff
@@ -26,4 +26,9 @@ class RemoteDataError(PandasError, IOError):
 if PANDAS_VERSION >= LooseVersion('0.14.0'):
     PANDAS_0140 = True
 else:
-    PANDAS_0140 = False
\ No newline at end of file
+    PANDAS_0140 = False
+
+def _init_session(session):
```

Comment:
Should use the logic as current ``_BaseReader``, and ``_BaseReader`` can use this definition.

https://github.com/pydata/pandas-datareader/blob/master/pandas_datareader/base.py#L57